config/config-example.js

```diff
@@ -378,4 +378,16 @@ exports.grouplist = [
 		symbol: ' ',
 		ip: 's',
 	},
+	{
+		name: 'Locked',
+		id: 'locked',
+		symbol: '‽',
+		punishgroup: true,
```

Comment:
Should possibly something like `punishment: "LOCK"` to relate it to the punishments.csv entry.